models/public/efficientnet-v2-b0/README.md

```diff
@@ -0,0 +1,84 @@
+# efficientnet-v2-b0
+
+## Use Case and High-Level Description
+
+The `efficientnet-v2-b0` model is a variant of the EfficientNetV2 pre-trained on ImageNet dataset for image classification task. EfficientNetV2 is a new family of convolutional networks that have faster training speed and better parameter efficiency than previous models. A combination of training-aware neural architecture search and scaling were used in the development to jointly optimize training speed and parameter efficiency.
+
+More details provided in the [paper](https://arxiv.org/pdf/2104.00298.pdf) and [repository](https://github.com/rwightman/pytorch-image-models).
```

Comment:
please use link on abstract instead of pdf document
```suggestion
More details provided in the [paper](https://arxiv.org/abs/2104.00298) and [repository](https://github.com/rwightman/pytorch-image-models).
```

Reply:
Fixed